ipysurfer/viz/surface.py

```diff
@@ -0,0 +1,162 @@
+from os import path as path
+
+from nibabel import freesurfer
+import numpy as np
+
+from .._utils import _check_units, _compute_normals, _get_subjects_dir
+
+
+class Surface:
+    """Container for a surface.
```

Comment:
Add a more verbose description of the class underneath the one-liner at the top of the docstring so people can understand in more detail what it does. (same goes for the "Brain" class)

Reply:
ok